docs/pages/setup/operations/ca-rotation.mdx

```diff
@@ -271,8 +282,17 @@ Host CA  initialized (mode: manual, started: Sep 20 01:44:36 UTC, ending: Sep 21
   type="warning"
   title="CA Pinning Warning"
 >
-  If you are using [CA Pinning](../admin/adding-nodes.mdx#untrusted-auth-servers) when adding new nodes, the CA pin will change after the rotation. Make sure you use the
-  *new* CA pin when adding nodes after rotation.
+  If you are using
+  [CA Pinning](../admin/adding-nodes.mdx#untrusted-auth-servers) when adding new
+  nodes, the CA pin will change after the rotation. Make sure you use the *new*
+  CA pin when adding nodes after rotation.
+</Admonition>
+
+<Admonition type="warning" title="Desktop Access">
```

Comment:
Is it poor form to have two `Adminition`s back to back like this?

Reply:
I've suggested moving the CA pinning warning since we use it twice in the page.